test/functionalTest/cases/501_cors/get_any_origin.test

```diff
@@ -0,0 +1,134 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+CORS GET any origin
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255 IPV4 -corsOrigin __ALL
+
+--SHELL--
+echo "01. Update/APPEND E1-T1-A1"
+echo "=========================="
+payload='{
+  "contextElements": [
+    {
+      "type": "T1",
+      "id":   "E1",
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "string",
+          "value": "1"
+        }
+      ]
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+echo "02. Query E1-T1"
+echo "==============="
+payload='{
+  "entities": [
+    {
+      "type": "T1",
+      "id": "E1"
+    }
+  ]
+}'
+orionCurl --url /v1/queryContext --payload "${payload}" --json
+echo
+echo
+
+--REGEXPECT--
+01. Update/APPEND E1-T1-A1
+==========================
+HTTP/1.1 200 OK
+Content-Length: 383
+Access-Control-Allow-Origin: *
```

Comment:
So you mean that `Access-Control-Allow-Origin` shouldn't be included in this case? In that case, I understand that the new code added to restReply.cpp should check that the request method was GET and only add the header in that case (I think you can get the method from the `ciP` object, maybe @kzangeli could provide more info on this).